Add the required steps to remove an already configured VCS repo within a workspace (#914)
hashicorp/terraform-website: content/source/docs/cloud/workspaces/settings.html.md

### VCS Connection and Repository

> You can use the "Select a VCS connection" buttons and "Repository" field to change which VCS repository the workspace gets configurations from. If you need to remove an already configured "Repository" you need to click on "Change VCS connection" and then select "skip this step". See also:

Reviewer: ```suggestion
You can use the "Select a VCS connection" buttons and "Repository" field to change which VCS repository the workspace gets configurations from. To remove an already configured VCS connection, use the "skip this step" link instead of one of the VCS buttons. See also:
```
